app/keymaps/linux.json

```diff
@@ -0,0 +1,32 @@
+{
+  "window:devtools":"Ctrl+Shift+I",
+  "window:reload":"Ctrl+Shift+R",
+  "window:reloadFull":"Ctrl+Shift+F5",
+  "window:preferences":"Ctrl+,",
+  "zoom:reset":"Ctrl+0",
+  "zoom:in":"Ctrl+plus",
+  "zoom:out":"Ctrl+-",
+  "window:new":"Ctrl+Shift+N",
+  "window:minimize": "Ctrl+M",
+  "window:full": "F11",
+  "window:close":"Ctrl+Shift+W",
+  
+  "tab:new":"Ctrl+Shift+T",
+  "tab:next":"Ctrl+Tab",
+  "tab:prev":"Ctrl+shift+Tab",
+  "pane:next":"Ctrl+Pageup",
+  "pane:prev":"Ctrl+Pagedown",
+  "pane:vertical":"Ctrl+Shift+D",
+  "pane:horizontal":"Ctrl+Shift+E",
+  "pane:close":"Ctrl+Shift+W",
+  
+  "editor:undo":"Ctrl+Shift+Z",
+  "editor:redo":"Ctrl+Shift+Z",
```

Comment:
I guess this should change to a different key. Ctrl+Y is regularly used as redo, so Ctrl+Shift+Y? Same for the Windows keymap.